apis/management.cattle.io/v3/authz_types.go

```diff
@@ -91,6 +90,15 @@ type PodSecurityPolicyTemplate struct {
 	Spec        extv1.PodSecurityPolicySpec `json:"spec,omitempty"`
 }
 
+type PodSecurityPolicyTemplateProjectBinding struct {
+	types.Namespaced
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	PodSecurityPolicyTemplateID string `json:"podSecurityPolicyTemplate" norman:"required"`
```

Comment:
To follow the established pattern, the name of the field should be `PodSecurityPolicyTemplateName` the json field shoudl be `PodSecurityPolicyTemplateId` and it should have `type=reference[podSecurityPolicyTemplate]` in the norman tag